Src/Test.UnitTest.Sarif.PatternMatcher.Function/Test.UnitTest.Sarif.PatternMatcher.Function.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(MSBuildThisFileDirectory)..\..\Targets\build.app.props" />
+  <Import Project="$(MSBuildThisFileDirectory)..\..\Targets\build.test.props" />
+
+  <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="5.10.3" />
+    <PackageReference Include="Moq" Version="4.15.2" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Plugins\Security\Security.csproj" />
+    <ProjectReference Include="..\Sarif.PatternMatcher.Function\Sarif.PatternMatcher.Function.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Include="..\Plugins\Tests.Security\TestData\SecurePlaintextSecrets\Inputs\VariousPats.txt" Link="TestData\VariousPats.txt" CopyToOutputDirectory="PreserveNewest" />
+  </ItemGroup>
+  
+  <Target Name="CopySecFilesAfterBuild" AfterTargets="AfterBuild">
+    <ItemGroup>
+      <SecurityFiles Include="$(OutDir)SEC*.*" />
```

Comment:
><SecurityFiles Include="$(OutDir)SEC*.*" /> [](start = 6, length = 43)

i think we are missing the txt files. try to keep the same tag name between projects

Reply:
SEC*.* includes all files required
![image](https://user-images.githubusercontent.com/76094515/104240643-409adc00-5411-11eb-9e3a-682bb0f99161.png)
I can update another project's copying target. What do you think?